src/components/PopupMenu/PopupMenu.tsx

```diff
@@ -16,7 +16,7 @@ export const PopupMenu = forwardRef<HTMLDivElement, PopupMenuProps>(
     ref
   ) {
     const clickInsideRef = useRef<any>()
-    const anchorRef = useRef<HTMLElement>()
+    const anchorRef = useRef<HTMLDivElement>()
```

Comment:
Theoretically the anchor could be something other than a div, like a button for example

Reply:
good callout, will fix